ext/tidy/tidy.c

```diff
@@ -1785,12 +1785,15 @@ static TIDY_NODE_METHOD(isComment)
 static TIDY_NODE_METHOD(isHtml)
 {
 	TIDY_FETCH_ONLY_OBJECT;
-
-	if (tidyNodeGetType(obj->node) & (TidyNode_Start | TidyNode_End | TidyNode_StartEnd)) {
-		RETURN_TRUE;
+	int type;
+	switch (type = tidyNodeGetType(obj->node)) {
```

Comment:
The assignment to type (and the variable) don't seem necessary.

Reply:
Oops! Debug code removed.